### ISSUE TITLE
Copy & Move Entire Collection or Folder to another Workspace

### ISSUE THREAD

**Issue by danurahadi** [a85/POSTMan-Chrome-Extension#11683]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

I think it will be easier for Developers to start a new workspace and want to copy / move some collections & folders from previous project / workspace. 

### Describe the solution you'd like

Simply add a new Copy & Move feature for each Collection and Folder that can copy / move them to another workspace

**Comment by giridharvc7** [comment]
@danurahadi thank you for reporting this, we'll track this as a feature request. 
Currently you can perform this by duplicating a collection and moving it to another workspace.

**Comment by Faq** [comment]
YEah this would ease a lot. Still cant believe something like this not exists. Even exporting folder with requests in it is not possible.

**Comment by danurahadi** [comment]
@Faq yap. Export a folder to JSON file would be a nice feature. Also, I think when add a new workspace User can choose 1 or more collections from existing workspaces, to be added automatically to it.

@giridharvc7 any good news for this request?
